snmp/datadog_checks/snmp/config.py

```diff
@@ -297,9 +297,49 @@ def add_uptime_metric(self):
             return
         # Reference sysUpTimeInstance directly, see http://oidref.com/1.3.6.1.2.1.1.3.0
         uptime_oid = OID('1.3.6.1.2.1.1.3.0')
-        self.scalar_oids.append(uptime_oid)
+        self.oid_config.add_parsed_oids([uptime_oid], [], [])
         self._resolver.register(uptime_oid, 'sysUpTimeInstance')
 
         parsed_metric = ParsedSymbolMetric('sysUpTimeInstance', forced_type='gauge')
         self.parsed_metrics.append(parsed_metric)
         self._uptime_metric_added = True
+
+
+class OIDConfig(object):
+    """
+    Manages scalar/next/bulk oids.
+    """
+
+    def __init__(self):
+        # type: () -> None
+        self._parsed_scalar_oids = []  # type: List[OID]
+        self._parsed_next_oids = []  # type: List[OID]
+        self._parsed_bulk_oids = []  # type: List[OID]
```

Comment:
(Nit) I think the naming verbosity can be reduced a bit...

```suggestion
        self._scalar = []  # type: List[OID]
        self._next = []  # type: List[OID]
        self._bulk = []  # type: List[OID]
```